README.md

```diff
@@ -8,6 +8,8 @@ For documenting and discussing the architecture of the VA.gov Platform.
 - [ADR Process](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/blob/main/adr/README.md)
 - [RFC Process](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/blob/main/rfc/README.md)
 
+## Security
+- Please do not share details that could provide a threat actor with architectural or technical insights that could be used in an attack scenario.
```

Comment:
Is there any way that we can define this further? Theoretically "we use AWS" could be used by an attacker, but it is publicly disclosable.

Reply:
Added.